src/main/java/frc/team3388/robot/RobotMap.java

```diff
@@ -1,4 +1,7 @@
 package frc.team3388.robot;
 
 public class RobotMap {
+    public static final int SHOOTER_CONTROLLER = 0;
+    public static final int DOUBLE_SOLENOID_1 = 1;
+    public static final int DOUBLE_SOLENOID_2 = 2;
```

Comment:
So, if you do create this factory method in `Shooter` then you can make these constants private, so there will be no need for `RobotMap`